### PR TITLE
feat: add exhaustive health check

### PR DESCRIPTION
**Pull request by pritamstyz4ever**
## Context

Health check does not include checking for queue and store

## Objective

This PR adds a check for queue and store health if request has exhaustive in query params

## License
I confirm that this contribution is made under a BSD license and that I have the authority necessary to make this contribution on behalf of its copyright owner.

### CODE REVIEW

File: plugins/status.js [hunk 1]
```diff
@@ -14,7 +50,21 @@ const statusPlugin = {
...
             method: 'GET',
             path: '/status',
-            handler: (_, h) => h.response('OK').code(200),
+            handler: async (request, h) => {
+                const { exhaustive } = request.query;
+
+                if (exhaustive) {
+                    const queueResponseCode = await invoke(`${request.server.app.ecosystem.queue}/v1/status`);
+                    if (queueResponseCode !== 200) {
+                        return h.response('SD Queue Service Unavailable').code(queueResponseCode);
```
Comment (jithine) [inline plugins/status.js:59]: why not fetch store response as well and return both in case of error

File: plugins/status.js [hunk 2]
```diff
@@ -14,7 +51,29 @@ const statusPlugin = {
         server.route({
             method: 'GET',
             path: '/status',
-            handler: (_, h) => h.response('OK').code(200),
+            handler: async (request, h) => {
+                const { exhaustive } = request.query;
```
Comment (tkyi) [inline plugins/status.js:55]: Wondering if there's a different term we can use vs `exhaustive`. Seems easy to misspell.
No strong preference though

Comment (jithine) [reply plugins/status.js:55]: extended ?

Comment (tkyi) [reply plugins/status.js:55]: I guess either `complete` for ease of use or just pick the word that best describes the intent (which could be `exhaustive` or `extensive` or whatever)

Comment (jithine) [reply plugins/status.js:55]: let's stick with `exhaustive` for now

**Review by jithine — APPROVED** [review]

**Pull request merged**